plutus-benchmark/bls-benchmarks/src/PlutusBenchmark/BLS12_381/Common.hs

```diff
@@ -274,7 +276,7 @@ groth16Verify
     :: BuiltinByteString  -- G1
     -> BuiltinByteString  -- G2
     -> BuiltinByteString  -- G2
-    -> BuiltinByteString  -- G1
+    -> BuiltinByteString  -- G2
```

Comment:
Seems like it might be worth having some `newtype`s here idk?

Reply:
Yeah, I was a bit confused when I looked at that today.  These types are wrapped in `newtypes` elsewhere in this file, but not here.  I think I did that because what we're really wanting to measure here is what happens on the chain, and having newtypes here would cost a little more.  I think we get `delay` or something in the uplc if we do that, which I was slightly surprised by when I found out.  I could be wrong about that though: I'll check.